### PR TITLE
Course search

### PR DESCRIPTION
**Pull request by Pelps12**
## Overview

This PR ads course titles to the search of courses

### CODE REVIEW

**Review by akevinge — CHANGES_REQUESTED** [review]
Your code looks solid. Thanks for working on this issue. Just a few things I want to point out.

- Ideally, we don't want the toggle. People should be able to type in a course title or catalog code and the correct courses will come up.
- You could use a library for this instead. I'd recommend [FuseJS](https://fusejs.io/), but you can use whatever works.

File: src/components/planner/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -32,14 +37,42 @@ function CourseSelectorContainer({
 
   const { data, isLoading } = q;
 
-  const { results, updateQuery } = useSearch({
+  React.useEffect(() => {
```
Comment (ZzRanger) [inline src/components/planner/Sidebar/Sidebar.tsx:40]: What is the purpose of this useEffect?

**Review by ZzRanger — DISMISSED** [review]
Tested it and LGTM! I'll merge this later tonight if no other problems come up.

File: src/components/planner/Sidebar/Sidebar.tsx (hunk 1 above)
Comment (Pelps12) [reply src/components/planner/Sidebar/Sidebar.tsx:40]: It's useless if not inefficient. It was to test Fusejs

Comment (Pelps12) [reply src/components/planner/Sidebar/Sidebar.tsx:40]: Meant to leave my comment here

**Pull request merged**